seed/challenges/01-front-end-development-certification/basic-javascript.json

```diff
@@ -2094,28 +2065,64 @@
       "challengeType": 1
     },
     {
+      "id": "bd7123c9c441eddfaeb5bdef",
+      "title": "Understanding Boolean Values",
+      "description": [
+        "Another data type is the <dfn>Boolean</dfn>. <code>Booleans</code> may only be one of two values: <code>true</code> or <code>false</code>. They are basically little on-off switches, where <code>true</code> is and <code>false</code> is \"off.\"  These two states are mutually exclusive.",
```

Comment:
`where <code>true</code> is and`... -> `where <code>true</code> is` **"on"** `and`...